lib/html/template/rose-bush/job-entry.html

```diff
@@ -1,139 +1,178 @@
 <div class="row-fluid entry"><!-- entry row -->
-<div class="span4">
-<!-- entry: cycle, name, submit_num, files -->
-<!-- entry: status -->
-{% if entry.status and "fail" in entry.status -%}
-<span class="label label-important">
-<i class="icon-warning-sign icon-white" title="{{entry.status}}"></i>
-</span>
-{% if entry.status != "fail" and entry.status != "fail(ERR)" -%}
-<span class="label label-important">
-{{entry.status[5:-1]}}
-</span>
-{% endif -%}
-{% elif entry.status == "success" -%}
-<span class="label label-success">
-<i class="icon-ok icon-white" title="{{entry.status}}"></i>
-</span>
-{% elif entry.status == "init" -%}
-<span class="label label-info">
-<i class="icon-play icon-white" title="{{entry.status}}"></i>
-</span>
-{% elif entry.status -%}
-<span class="label">
-<i class="icon-info-sign" title="{{entry.status}}"></i>
-</span>
-{% endif -%}
-<!-- entry: status -->
-{% if entry.cycle != "1" -%}
-<a href="{{script}}/jobs/{{user}}/{{suite}}?cycles={{entry.cycle|replace('+', '%2B')}}" class="cycle" title="{{entry.cycle}}">{{entry.cycle}}</a>
-{% endif -%}
-<a href="{{script}}/jobs/{{user}}/{{suite}}?tasks={{entry.name}}"
-title="{{entry.name}}">{{entry.name}}</a>
-{% if entry.submit_num_max == 1 -%}
-{% elif entry.submit_num == entry.submit_num_max -%}
-<span class="badge badge-inverse"
-title="submit {{entry.submit_num}}">{{entry.submit_num}}</span>
-{% else -%}
-<span class="badge"
-title="submit {{entry.submit_num}}">{{entry.submit_num}}</span>
-{% endif -%}
-</div>
+  <div class="span4">
+    <!-- entry: cycle, name, submit_num, files -->
+    <!-- entry: status -->
+    {% if entry.status and "fail" in entry.status -%}
+    <span class="label label-important">
+      <i class="icon-warning-sign icon-white" title="{{entry.status}}"></i>
+    </span>
+    {% if entry.status != "fail" and entry.status != "fail(ERR)" -%}
```

Comment:
This if needs indent nesting as its associated endif looks in error against the elif.